webservices/ph5tostationxml.py

```diff
@@ -154,36 +150,29 @@ def __init__(self, args):
         #    '/opt/k4/apps/pn4/webservices/RESP.XX.NR021..HHZ.130.32.100')
         
         nickname = args.get('nickname')
-        if nickname[-3:] == 'ph5':
-
-            PH5FILE = os.path.join(args.get('ph5path'), args.get('nickname'))
-
-        else:
-
-            PH5FILE = os.path.join(args.get('ph5path'), args.get('nickname') + '.ph5')
-

```

Comment:
I removed these lines since `PH5FILE` was not referenced anywhere else in the code.